core/rawdb/accessors_chain.go

```diff
@@ -64,6 +64,26 @@ func DeleteCanonicalHash(db ethdb.KeyValueWriter, number uint64) {
 	}
 }
 
+func IsTrustBlock(db ethdb.Reader, hash common.Hash) bool {
+	data, _ := db.Get(trustBlockHashKey(hash))
+	if len(data) == 0 {
+		return false
+	}
+	return bytes.Equal(data,[]byte{byteTrue})
+}
+
+func MarkTrustBlock(db ethdb.KeyValueWriter, hashkey common.Hash) {
+	if err := db.Put(trustBlockHashKey(hashkey),[]byte{byteTrue}); err != nil {
+		log.Crit("Failed to store trust block hash")
+	}
+}
+
+func DeleteTrustBlockHash(db ethdb.KeyValueWriter, hash common.Hash) {
```

Comment:
Never been used, I think we should delete the storage once we are 100% sure it will not be used any longer.